util/memory/tracker.go

```diff
@@ -396,11 +408,66 @@ func (t *Tracker) BufferedConsume(bufferedMemSize *int64, bytes int64) {
 	}
 }
 
+// Release is used to release memory tracked, track the released memory until GC triggered if needed
```

Comment:
Could you described that the proper way and situation to use the `Release` function. And I think `fake obj ref` is a good and also a hack implementation, so please also explain how it works in comment.